.gitattributes

```
@@ -0,0 +1,9 @@
+dependencies.md                                                  linguist-generated=true
+doc/changes/changelog.md                                         linguist-generated=true
+pk_generated_parent.pom                                          linguist-generated=true
+.github/workflows/broken_links_checker.yml                       linguist-generated=true
+.github/workflows/ci-build-next-java.yml                         linguist-generated=true
```

```suggestion
.github/workflows/ci-build.yml                                   linguist-generated=true
.github/workflows/ci-build-next-java.yml                         linguist-generated=true
```

ci-build.yml is also generated